src/sss_client/ssh/sss_ssh_knownhostsproxy.c

```diff
@@ -17,8 +17,33 @@
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
+#include "config.h"
 
 #include <stdio.h>
+#include <stdlib.h>
+
+
+#ifndef BUILD_SSH_KNOWN_HOSTS_PROXY
+
+static const char message[] =
+    "\n"
+    "******************************************************************\n"
+    "You are using the deprecated tool sss_ssh_knownhostsproxy.\n"
+    "Please read the documentation to migrate to sss_ssh_knownhosts(1).\n"
+    "******************************************************************\n"
+    "\n";
+
+
+int main(void)
+{
+    fprintf(stderr, message);
```

Comment:
`fprintf` -> `ERROR`

Reply:
Hm, I'm not sure if localization software is smart enough to pull message from pointer... (needs to be checked)

Reply:
Hi,

you have to wrap the original string with `_(...)`, we do similar e.g. in https://github.com/SSSD/sssd/blob/master/src/tools/common/sss_tools.c#L55.

HTH

bye,
Sumit

Reply:
Done

Reply:
Now it is really solved.